src/EnliteMonolog/Service/MonologServiceFactory.php

```diff
@@ -202,7 +202,39 @@ public function createProcessor($container, $processor)
             }
         }
 
-        throw new RuntimeException('Unknown processor type, must be a Closure or the FQCN of an invokable class');
+        if (is_array($processor)) {
+            if (!isset($processor['name'])) {
+                throw new RuntimeException('Cannot create logger processor');
+            }
+
+            $processorClassName = $processor['name'];
+
+            if (!class_exists($processorClassName)) {
+                throw new RuntimeException('Cannot create logger processor (' . $processorClassName . ')');
+            }
+
+            $arguments = array_key_exists('args', $processor) ? $processor['args'] : array();
+
+            if (!is_array($arguments)) {
+                throw new RuntimeException('Arguments of processor (' . $processorClassName . ') must be array');
+            }
+
+            try {
+                /** @var FormatterInterface $instance */
+                $instance = $this->createInstanceFromArguments($processorClassName, $arguments);
+            } catch (\InvalidArgumentException $exception) {
+                throw new RuntimeException(sprintf(
+                    'Processor(%s) has an invalid argument configuration',
+                    $processorClassName
+                ), 0, $exception);
+            }
+
+            if (is_callable($instance)) {
+                return $instance;
+            }
+        }
+
+        throw new RuntimeException('Unknown processor type, must be a Closure, array or the FQCN of an invokable class');
```

Comment:
the linter fails on this line's length, since `, array` was added. try moving the message string to its own line. you can run `composer lint:style` to check this before waiting for the lengthy build to complete.

Reply:
done